packages/playground/src/samples/examples.ts

```diff
@@ -1,6 +1,8 @@
 import { Sample } from './Sample';
+import validator from '@rjsf/validator-ajv8';
```

Comment:
```suggestion
import validator from '@rjsf/validator-ajv8';

import { Sample } from './Sample';
```

Reply:
@heath-freenome how was this passing the linter before if `validator` was imported but not being used?

Reply:
?! good question Thanks for fixing